test/resources/verify_metrics/noncompliant.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: policy.open-cluster-management.io/v1
 kind: Policy
 metadata:
-  name: policy-verify-metrics-noncompliant
+  name: policy-vm-non
```

Comment:
I'm not a huge fan of using nonstandard acronyms like this, but there are a lot of them at this point from the global-set namespace update and they do shorten the names a fair bit, so we can leave them as-is.